docs/user-guide/docker-cli-to-kubectl.md

```diff
@@ -0,0 +1,255 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<h1>PLEASE NOTE: This document applies to the HEAD of the source
+tree only. If you are using a released version of Kubernetes, you almost
+certainly want the docs that go with that version.</h1>
+
+<strong>Documentation for specific releases can be found at
+[releases.k8s.io](http://releases.k8s.io).</strong>
+
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+![WARNING](http://kubernetes.io/img/warning.png)
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+# kubectl for docker users
+
+In this doc, we introduce the kubernetes command line to for interacting with the api to docker-cli users. The tool, kubectl, is designed to be familiar to docker-cli users but there are a few necessary differences. Each section of this doc highlights a docker subcommand explains the kubectl equivalent.
+
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [kubectl for docker users](#kubectl-for-docker-users)
+      - [docker run](#docker-run)
+      - [docker ps](#docker-ps)
+      - [docker exec](#docker-exec)
+      - [docker logs](#docker-logs)
+      - [docker stop and docker rm](#docker-stop-and-docker-rm)
+      - [docker login](#docker-login)
+      - [docker version](#docker-version)
+      - [docker info](#docker-info)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+#### docker run
+
+How do I run an nginx container and expose it to the world? Checkout [kubectl run](kubectl/kubectl_run.md).
+
+With docker:
+```
+$ docker run -d --name nginx-app -p 80:80 nginx
```

Comment:
It is not the default.  And even if you specify always, it is silently
ignored if you use -d.

On Wed, Jul 15, 2015 at 2:52 PM, Brian Grant <notifications@github.com>
wrote:

> In docs/user-guide/docker-cli-to-kubectl.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11271#discussion_r34734116>
> :
>
> > +      - [docker exec](#docker-exec)
> > +      - [docker logs](#docker-logs)
> > +      - [docker stop and docker rm](#docker-stop-and-docker-rm)
> > +      - [docker login](#docker-login)
> > +      - [docker version](#docker-version)
> > +      - [docker info](#docker-info)
> > +
> > +<!-- END MUNGE: GENERATED_TOC -->
> > +
> > +#### docker run
> > +
> > +How do I run an nginx container and expose it to the world? Checkout [kubectl run](kubectl/kubectl_run.md).
> > +
> > +With docker:
> > +```
> > +$ docker run -d --name nginx-app -p 80:80 nginx
>
> I assume --restart-always is not the default. @erictune
> <https://github.com/erictune> had a good point that this is what kubectl
> run effectively does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11271/files#r34734116>
> .
>